pkg/node/node.go

```diff
@@ -16,26 +16,34 @@ package node
 
 import (
 	"net"
+	"path"
 
 	"github.com/cilium/cilium/api/v1/models"
+	"github.com/cilium/cilium/pkg/defaults"
 	"github.com/cilium/cilium/pkg/option"
 
 	"k8s.io/api/core/v1"
 )
 
 // Identity represents the node identity of a node.
 type Identity struct {
-	Name string
+	Name    string
+	Cluster string
 }
 
 // String returns the string representation on NodeIdentity.
 func (nn Identity) String() string {
-	return nn.Name
+	return path.Join(nn.Cluster, nn.Name)
```

Comment:
Is `String()` being used for any for of identification of this node in the kvstore? If yes, will this break the ability to do a downgrade?

Reply:
It was used internally to identify but we don't have any APIs that depend on it.